guardian/mixins.py

```diff
@@ -138,6 +144,7 @@ class SecureView(PermissionRequiredMixin, View):
     return_403 = False
     return_404 = False
     raise_exception = False
+    permission_denied_message = None
```

Comment:
We can make this be blank str `''`, and it would help with the `get_permission_denied_message` function to be simple.